src/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Awesome Books with ES6</title>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update your project title by adding the correct one as your application is not named `Awesome Books with ES6` but `Leaderboard`. This will help users to have an idea about your application.

Reply:
Thanks a lot !